tests/kit/smooth_1D.py

```diff
@@ -0,0 +1,34 @@
+"""Test kit.smooth_1D."""
+
+
+# --- import --------------------------------------------------------------------------------------
+
+
+import numpy as np
+
+import WrightTools as wt
+
+
+# --- test ----------------------------------------------------------------------------------------
+
+
+def test_1():
+    # create arrays
+    x = np.linspace(0, 10, 1000)
+    y = np.sin(x)
+    r = np.random.rand(1000) - .5
```

Comment:
We should set the seed to avoid another situation like #553 where on occasion, but rarely, it fails.